js/join.js

```diff
@@ -126,32 +135,36 @@ function showPage(currentFlowState) {
 //Validators
 function arePersonalDetailsValid() {
   return (
-    state.value.trim().length > 3 &&
-    city.value.trim().length > 3 &&
-    country.value.trim().length > 3
+    state.value.trim().length >= 3 &&
+    city.value.trim().length >= 3 &&
+    country.value.trim().length >= 3
   );
 }
 
 function introPageValidator() {
   return (
-    introduction.value.trim().split(' ').length > 100 &&
-    skills.value.trim().split(' ').length > 6 &&
-    college.value.trim().split(' ').length > 5 &&
-    forFun.value.trim().split(' ').length > 100 &&
-    funFact.value.trim().split(' ').length > 100
+    introduction.value.trim().split(' ').length >= 100 &&
+    skills.value.trim().split(' ').length >= 5 &&
+    college.value.trim().split(' ').length >= 5 &&
+    forFun.value.trim().split(' ').length >= 100 &&
+    funFact.value.trim().split(' ').length >= 100
   );
 }
 
 function whyRdsPageValidator() {
-  return whyRds.value.trim().split(' ').length > 100 && foundFrom.value != '';
+  return whyRds.value.trim().split(' ').length >= 100 && foundFrom.value != '';
 }
 
 function dataValidator(element, size) {
-  if (element.value.trim().split(' ').length > size) {
-    element.classList.add('correct-data');
+  let counter = document.getElementById(element.id + 'Counter');
+  let words_left = size - element.value.trim().split(' ').length;
+  counter.innerText = `Atleast, ${words_left} more word(s) required`;
```

Comment:
Do we need the comma after "Atleast"? 
Is it grammatically correct?

Reply:
As per grammarly yes

Reply:
Cool